ci-operator/step-registry/vsphere/provision/bastionhost/tang/vsphere-provision-bastionhost-tang-ref.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "vsphere/provision/bastionhost/tang/vsphere-provision-bastionhost-tang-ref.yaml",
```

Comment:
Most of implementation in the step is to generate manifest files, so maybe we  can rename it to `ipi-conf-tang`, also we can share the step with other platform in the future.

Reply:
updated